docs/_embed/solutions/partner-api.md

```diff
@@ -349,14 +352,50 @@ Pragma: no-cache
   "refresh_token": "9D9oz2ZzaouT12LpvklQwNBf6s4vva"
 }
 ```
-🎉 This response contains the access token that you'll use to make API request on the user's behalf.
+🎉 This response contains the access token that you'll use to make API request on the user's behalf, as well as a
+refresh token which should be stored securely for later use.
```

Comment:
Can we clarify what it means to store something securely?

_Suggestion:_

> The refresh token MAY NOT be stored in localStorage
>
> The refresh token MAY be stored in a Secure; HTTPOnly cookie
>
> The refresh token MAY be stored in a server side database

Reply:
There's nothing we can do to force compliance, but it's extremely bad to put the refresh token in something that can be easily stolen by 3rd parties, and I think we want to hammer this point home.

Reply:
That sounds good. Do you think we should ask the same of the access token? Or is it okay for partners to keep doing whatever they were currently doing, now that it'll expire?

Reply:
I think we should give the same suggestion for the access token. It's just _even more important_ for the refresh token since that's something that can be used to obtain **infinite** access